ArcGISRuntimeSDKQt_QMLSamples/Scenes/RealisticLightingAndShadows/RealisticLightingAndShadows.qml

```diff
@@ -0,0 +1,151 @@
+// [WriteFile Name=RealisticLightingAndShadows, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.ArcGISRuntime 100.10
+
+Rectangle {
+    width: 800
+    height: 600
+
+    // Create a scene view
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+        atmosphereEffect: Enums.AtmosphereEffectRealistic
+        sunLighting: Enums.LightingModeLightAndShadows
+        sunTime: "2018-08-10T08:30"
+
+        Component.onCompleted: {
+            setViewpointCameraAndSeconds(camera, 0)
+        }
+
+        Camera {
+            id: camera
+            Point {
+                y: 45.54605153789073
+                x: -122.69033380511073
+                z: 941.0002111233771
+            }
+            heading: 162.58544227544266
+            pitch: 60.0
+            roll: 0
+        }
+
+
+        // create a scene, which is a default property of scene view
+        Scene {
+            // add a basemap
+            BasemapTopographic {}
+
+            // add a surface, which is a default property of scene
+            Surface {
+                // add an arcgis tiled elevation source...elevation source is a default property of surface
+                ArcGISTiledElevationSource {
+                    url: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+            }
+
+            //! [add a scene service with ArcGISSceneLayer]
```

Comment:
The `//! []` syntax is a only needed when we will be referencing the snippet from in some documentation (we have a mechanism for extracting ehse and pulling into the doc.

```suggestion
            // add a scene service with ArcGISSceneLayer
```